spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -156,6 +161,15 @@ export function connectToSpotTV(joinCode, shareMode) {
 
             history.push(ROUTES.CODE);
 
+            // Assumption alert: kicks can only happen when a temporary remote
+            // has been removed at the end of a meeting.
+            if (error === CONNECTION_EVENTS.KICKED) {
```

Comment:
Hmm reading this it doesn't sound generic enough. The kick logic is currently XMPP specific, so how about more generic event name? Something around the lines of connection closed by the server or temporary remote disconnected by the server?

Reply:
Changed to CLOSED_BY_SERVER. Let me know if it needs more changing.